static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
```

Comment:
I suppose we can, but up until now the strategy has been to reflect their data model in ours for the fields we didn't have yet.

If we want to go this route the current `phone_number` column on profile needs to get widened by enough chars that it can fit a 3-digit country code + whatever extra formatting might be present (probably the leading `+` and whatever delimeter falls between the country code and the rest of the number).

If you go this route, could you open a new ticket to fix the profile export to Pearson to address this change in data structuring?